pkgs/top-level/all-packages.nix

```diff
@@ -12797,6 +12797,11 @@ with pkgs;
 
   gnudatalanguage = callPackage ../development/interpreters/gnudatalanguage {
     inherit (llvmPackages) openmp;
+    # Workaround to make udunits build on Darwin
+    # Remove this after the platform changes are made
+    udunits = udunits.overrideAttrs (oldAttrs: {
+      meta.platforms = lib.platforms.all;
+    });
```

Comment:
This makes no sense. 

If `udunits.meta.platforms` exclude Darwin, there are two possibilities:

- it does not run in Darwin, therefore forcing it to run in Darwin will fail; therefore, GDL does not run in Darwin.
- it does run in Darwin, therefore the original udunits expression should be modified.

Reply:
Knocked it out after the udunits PR had been merged.